### PR TITLE
Breadcrumbs

### PR DESCRIPTION
**Pull request by brianbodollo**
image

### CODE REVIEW

**Comment by brianbodollo** [comment]
@srph

File: styles/components/_breadcrumbs.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  color: $brand-gray;
+  border-radius: $border-radius-base;
+  font-size: $mini;
+
+  > a { 
+    color: $brand-gray; 
+    text-decoration: none;
+  }
+
+  > a > li { display: inline-block; }
```
Comment (srph) [inline styles/components/_breadcrumbs.scss:13]: It should be the other way around lol (`li > a`) becuase it's always `ul > li`. Don't use `li` without `ul` or `ol`.

File: styles/components/_breadcrumbs.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+  a + a:before {
```
Comment (srph) [inline styles/components/_breadcrumbs.scss:15]: `li` :)

**Comment by brianbodollo** [comment]
refer to https://github.com/dev-onenetworkecommerce/aia/pull/143

**Pull request closed without merging**